Gemfile

```diff
@@ -8,6 +8,13 @@ group :development do
   gem 'pry'
 end
 
+if RUBY_VERSION < "2.0"
+  gem 'json', '< 2.0.0'
+  gem 'net-ssh', '<= 2.10'
```

Comment:
'~> 2.9.0' pins it better I think, there are a few things incompatible & warnings with 2.5.1